src/css/skills.scss

```diff
@@ -0,0 +1,100 @@
+@import './core/constants';
+@import './core/mixins';
+
+//skill container
+$skill-s: 145px;
+$skill-m: 217.5px;
+$skill-l: 290px;
+$skill-xl: 435px;
+
+h2 {
+  font-size: $font-xl;
+}
+
+#skills {
+  height: max-content;
+  position: relative;
+}
+
+.skills-heading {
+  padding-left: 245px;
```

Comment:
if you need to have such large padding, good chance that you over complicated things.
One easy way to do this is having a wrapper div that contains both the title and skills content, and use something like:
```
.wrapper {
  width: 940px; // might be different value, this is just a demo
  margin: 0 auto;
}
```